sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -663,14 +663,9 @@ class ResolveSessionCatalog(
   object TempViewOrV1Table {
     def unapply(nameParts: Seq[String]): Option[Seq[String]] = nameParts match {
       case _ if isTempView(nameParts) => Some(nameParts)
-      case SessionCatalogAndTable(_, tbl) =>
```

Comment:
I'd expect to see `case SessionCatalogAndTable(_, tbl) => Some(table)`, as this is `TempViewOrV1Table`

Reply:
Oh, it's already there in line 668.